Porting/perldocstyle.pod

```diff
@@ -0,0 +1,1065 @@
+=encoding utf8
+
+=head1 NAME
+
+perldocstyle - A style guide for writing Perl's documentation
+
+=head1 DESCRIPTION
+
+This document is a guide for the authorship and maintenance of the
+documentation that ships with Perl. This includes the following:
+
+=over
+
+=item *
+
+The several dozen manual sections whose filenames begin with "C<perl>",
+such as C<perlobj>, C<perlre>, and C<perlintro>. (And, yes, C<perl>.)
+
+=item *
+
+The documentation for all the modules included with Perl (as listed by
+L<C<perlmodlib>|perlmodlib>).
+
+=item *
+
+The hundreds of individually presented reference sections derived from
+the L<C<perlfunc>|perlfunc> file.
+
+=back
+
+This guide will hereafter refer to user-manual section files as I<man
+pages>, per Unix convention.
+
+=head2 Purpose of this guide
+
+This style guide aims to establish standards, procedures, and philosophies
+applicable to Perl's core documentation.
+
+Adherence to these standards will help ensure that any one part of
+Perl's manual has a tone and style consistent with that of any other. As
+with the rest of the Perl project, the language's documentation
+collection is an open-source project authored over a long period of time
+by many people. Maintaining consistency across such a wide swath of work
+presents a challenge; this guide provides a foundation to help mitigate
+this difficulty.
+
+This will help its readers--especially those new to Perl--to feel
+more welcome and engaged with Perl's documentation, and this in turn
+will help the Perl project itself grow stronger through having a larger,
+more diverse, and more confident population of knowledgeable users.
+
+=head2 Who this guide is for
```

Comment:
I raised this concern privately but:

Does CMOS not recommend “whom” as the object of a preposition?

Maybe reword to “Who should read this guide” …

Reply:
I've asked the editor I'm working with to weigh in on this one! Will update.

Reply:
The editor says:

> Essentially Felix is correct that the sentence should be "Whom" in this context, and you are correct that it doesn't sound right and is not normally written that way (mostly due to the ending preposition and also the way English audiences interpret what we read). So to quote my favourite copyediting book, "since being correct is only half the goal, it is perhaps best to revise" (The Copyeditor's  Handbook, Amy Einsohn).  
>
>Would the title, "Intended Audience" be too formal?

I think that's a great idea, and I'll make that suggested change.

Reply:
Along this line:

In light of “whom”’s increasingly esoteric status in common usage, it _may_ be worthwhile for the style guide to recommend that “whom” be avoided, and that phrases where it would otherwise be appropriate should be reworded. This very phrase is a case in point.